ruby_programming/basic_ruby/nested_collections.md

```diff
@@ -0,0 +1,330 @@
+### Introduction
+This lesson is going to build on your knowledge of arrays and hashes. As you have seen, arrays and hashes are great ways to store data. However, sometimes the data is more complex and needs more structure than a basic array or hash offers. 
+
+Let's take a look at how you can use nested arrays and nested hashes to store more complex data.
+
+### Learning Outcomes
+By the end of this lesson, you should be able to do the following:
+
+- Describe a nested array and hash.
+- Explain what data is useful to store in a nested array and hash.
+- Explain how to access data in a nested array and hash.
+- Explain why the `#dig` method is useful.
+- Explain how to add data to a nested array and hash.
+- Explain how to delete data in a nested array and hash.
+- Explain how to create a new nested array that is not mutable.
+- Explain how to iterate over a nested array and hash.
+
+### Nested Arrays
+Arrays can contain any type of data, including other arrays. An array that contains other arrays is called a nested array, or a multidimensional array.
+
+Nested arrays can be useful to store groups of similar data or positional data. The following nested array of test scores is storing groups of similar data and the teacher mailboxes is storing groups of positional data. I'm not sure if this is common around the world, but every school office that I can recall has teacher mailboxes in a paper organizer.
```

Comment:
What do you think about removing "I'm not sure if this is common around the world, but every school office that I can recall has teacher mailboxes in a paper organizer."

Imo it seems a little out of place and students should be able to get the jist of the examples :)

Reply:
I agree & will remove it.